examples/incompressible-flow/3d-periodic-hills/periodic-hills.prm

```diff
@@ -144,15 +144,12 @@ end
 
 subsection linear solver
   subsection fluid dynamics
-    set method                                = gmres
-    set max iters                             = 5000
-    set relative residual                     = 1e-3
-    set minimum residual                      = 1e-9
-    set max krylov vectors                    = 200
-    set preconditioner                        = ilu
-    set ilu preconditioner fill               = 0
-    set ilu preconditioner absolute tolerance = 1e-12
-    set ilu preconditioner relative tolerance = 1.00
-    set verbosity                             = verbose
+    set method             = gmres
+    set max iters          = 5000
+    set relative residual  = 1e-3
+    set minimum residual   = 1e-9
+    set max krylov vectors = 200
```

Comment:
Maybe we could lower the amount of iterations and increase the max number of krylov vectors to match it?

Reply:
Yes, I think this is a good idea. I have checked my output files, and I think 300 would be reasonable for both.